### PR TITLE
Update specification name to Vibration API

### PR DESCRIPTION
**Pull request by anssiko**
This name matches the latest published version.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vibration/pull/51.html" title="Last updated on Nov 29, 2024, 1:45 PM UTC (db625fd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vibration/51/d055733...db625fd.html" title="Last updated on Nov 29, 2024, 1:45 PM UTC (db625fd)">Diff</a>

### CODE REVIEW

**Comment by anssiko** [comment]
(We should wait for #52 to land before we merge this to test the workflow with this PR.)

**Review by himorin — APPROVED** [review]

**Comment by himorin** [comment]
update required also for previous publish and crend (esp. latter)

**Comment by anssiko** [comment]
Thanks for the review, I pushed an update to the metadata.

**Pull request merged**
